tests/test.py

```diff
@@ -176,7 +176,7 @@ def test_stack_wait_for_stack_not_done(self):
         with self.assertRaises(errors.CfnTimeoutError):
             print cloudformation.Cloudformation(
                 self.env.aws_profile).wait_for_stack_done(self.stack_name, 1, 1)
-        
+
```

Comment:
Can you remove this change from the PR - it's unrelated to the change we're making here and will just confuse git history